src/ccstruct/tabletransfer.h

```diff
@@ -0,0 +1,60 @@
+/******************************************************************************
+ * File:        tabletransfer.h
+ * Description: Infrastructure for the transfer of table detection results
+ * Author:      Sintun
```

Comment:
Is this an alias name? Any reason why you prefer not to write your real name here and also below?

Reply:
I'm indifferent, changed to real name.